Python/optimizer_cases.c.h

```diff
@@ -1368,9 +1368,16 @@
             break;
         }
 
-        /* _BEFORE_ASYNC_WITH is not a viable micro-op for tier 2 */
-
-        /* _BEFORE_WITH is not a viable micro-op for tier 2 */
+        case _LOAD_SPECIAL: {
+            _Py_UopsSymbol *attr;
+            _Py_UopsSymbol *self_or_null;
+            attr = sym_new_not_null(ctx);
+            self_or_null = sym_new_not_null(ctx);
```

Comment:
This doesn’t seem right, it can push null?

Reply:
Yes, the default is `sym_new_not_null` as only one or two instructions push NULL. We'll need to be explicit here.